content/docs/release-notes.md

```diff
@@ -2,7 +2,60 @@
 title: Release Notes
 ---
 
-## 2022-07-11
+## 2022-07-19 Storage release
+
+### What's new
+
+* Safekeeper: Back up WAL to S3 for disaster recovery.
+
+* Safekeeper: Download WAL from S3 on demand.
+
+* Propagate postgres authentication errors to the user.
+
+* Rename `contrib/zenith` to `contrib/neon`.
+* Rename custom GUCs:
+    * `zenith.page_server_connstring` -> `neon.pageserver_connstring`
+    * `zenith.zenith_tenant` -> `neon.tenant_id`
+    * `zenith.zenith_timeline` -> `neon.timeline_id`
+    * `zenith.max_cluster_size` -> `neon.max_cluster_size`
+
+* Rename `zenith_admin` role to `cloud_admin`
```

Comment:
```suggestion
* Control plane: Rename `zenith_admin` role to `cloud_admin`
```

or is it too early to introduce control plane to our release notes? WDYT, @kelvich @ololobus ?